main.js

```diff
@@ -0,0 +1,49 @@
+const formDataDisplay = document.querySelector("#formData");
+console.log(formDataDisplay);
+const form = document.querySelector("#form");
+form.addEventListener("submit", (event) => {
+  event.preventDefault();
+
+  const formData = new FormData(form);
+
+  formData.forEach((value, key) => {
+    console.log(key, value);
+  });
+});
+
+function createListItem() {
+  const list = document.getElementById("list");
+
+  const listItem = document.createElement("div");
+  listItem.id = "listItem";
+  //h2 or Name
+  const h2 = document.createElement("h2");
+  const textNode = document.createTextNode("3.Vaishnavi Nikam");
+  h2.appendChild(textNode);
+  listItem.appendChild(h2);
+  // Email
+  const a = document.createElement("a");
+  a.href = "mailto:vaishnavinikam36@gmail.com";
+  const emailTextNode = document.createTextNode("vaishnavinikam36@gmail.com");
```

Comment:
this is good naming
use similar naming in other places as well
e.g. `nameElement`, `nameTextNode`, `genderElement`, `employmentStatusElement`, `employmentStatusTextNode` etc.

Reply:
`emailElement`, `notesElement`, `notesTextNode` are remaining